hydra-node/test/Hydra/Model.hs

```diff
@@ -236,7 +194,7 @@ instance StateModel WorldState where
       Open{} -> genNewTx
       _ -> genSeed
    where
-    genSeed = Some . Seed <$> resize 7 partyKeys
+    genSeed = Some . Seed <$> resize 3 partyKeys
```

Comment:
You could use the already hard-coded `maximumNumberOfParties` here. Currently it's `4`